Trimble.Modus.Components/Styles/DarkThemeColors.xaml

```diff
@@ -30,6 +30,9 @@
   <Color x:Key="SecondaryLight">#B7B9C3</Color>
   <Color x:Key="SecondaryDark">#585C65</Color>
 
+  <Color x:Key="IconLight">#252A2E</Color>
```

Comment:
Do we need two colors in both the Dark adn Light theme colors xaml file? @Sofiya-kumar Can you cross check this?

Reply:
Yes we need to add both dark and light theme colors xam file if we add any new colors